README.md

```diff
@@ -1 +1,37 @@
-# auth-api
\ No newline at end of file
+# auth-api
+
+by Tahmina Ringer
+
+## Links/Resources
+
+- [GitHub PR](https://github.com/Tahmina-Ringer-401-advanced-javascript/bearer-auth/pull/2)
+- [Heroku-dashboard](https://dashboard.heroku.com/apps/bearer-auth-tahmina)
+- [Heroku-deployed](https://bearer-auth-tahmina.herokuapp.com/)
+
+- Resource[jsonwebtoken](https://www.npmjs.com/package/jsonwebtoken)
+
+Basic express server using modularization, middleware and tests.
+`npm i` express, MongoDB, Mongoose, base-64, bcrypt, cors, dotenv, jsonwebtoken, mongoose, morgan and jest.
+
+## Setup
+
+- **.env Requirements**
+  - PORT=3000
+  - MONGOOSE_URI=mongodb://localhost:27017/auth
+  - SECRET=SECRET
+
+## Running the App
+
+- npm start or nodemon
+
+- **Endpoint**
+  - used postman
+
+## Test
+
+- run command `npm test`
+
+## UML
+
+- created with AWW - A Web Whiteboard
+- [UML](UML.png)
```

Comment:
Everything looks really great Tahmina, I would just like to see tests added. If you focus on testing the v1 or v2 routes that would be great!